django/db/migrations/migration.py

```diff
@@ -184,15 +183,24 @@ def suggest_name(self):
         name to avoid VCS conflicts if possible.
         """
         name = None
-        if len(self.operations) == 1:
-            name = self.operations[0].migration_name_fragment
-        elif (
-            len(self.operations) > 1 and
-            all(isinstance(o, operations.CreateModel) for o in self.operations)
-        ):
-            name = '_'.join(sorted(o.migration_name_fragment for o in self.operations))
-        if name is None:
-            name = 'initial' if self.initial else 'auto_%s' % get_migration_name_timestamp()
+        if self.initial:
+            return "initial"
+
+        raw_fragments = (op.migration_name_fragment for op in self.operations)
+        fragments = [name for name in raw_fragments if name]
+
+        if not fragments:
+            return f"auto_{get_migration_name_timestamp()}"
```

Comment:
f-strings shouldn't contain function calls. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).